contracts/protocol/bases/ProtocolBase.sol

```diff
@@ -308,6 +308,40 @@ abstract contract ProtocolBase is PausableBase, ReentrancyGuardBase {
         exists = (_sellerId > 0 && seller.id == _sellerId);
     }
 
+    /**
+     * @notice Fetches a given seller pending update from storage by id
+     *
+     * @param _sellerId - the id of the seller
+     * @return exists - whether the seller or auth token pending update exists
+     * @return sellerPendingUpdate - the seller pending update details. See {BosonTypes.Seller}
+     * @return authTokenPendingUpdate - auth token pending update details
+     */
+    function fetchSellerPendingUpdate(uint256 _sellerId)
```

Comment:
Does this really need to live in ProtocolBase? Could it instead go in SellerBase? Seems a bit specific to this use case to need to be inherited by every facet.

Reply:
moved